AdminDashboard/AdminDashboard.csproj

```diff
@@ -1,17 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <UserSecretsId>ecc97d97-957d-4b2e-86b6-c0d61154714e</UserSecretsId>
   </PropertyGroup>
-
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
-
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.0.0" />
     <PackageReference Include="MySql.Data.EntityFrameworkCore" Version="8.0.18" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc4" />
```

Comment:
For simplicity, I've removed Swagger from this project now that the server-side should be mostly done. It was only there for testing purposes!